src/base.py

```diff
@@ -17,18 +17,27 @@
 
 
 def change_to_base(i,b):
-        if i== 0:
-            return [0]
-        else:
-            number_in_b=[]
-            while i>0:
-                q=(i//b)
-                r=i%b
-                number_in_b.append(r)
-                i=q
-            number_in_b.reverse()
-            better_format = int("".join(map(str, number_in_b)))
-            print(better_format)
-
-change_to_base(10000,13)
+    if i== 0:
+        return [0]
+    if i>0:
+        number_in_b=[]
+        while i>0:
+            q=(i//b)
+            r=i%b
+            number_in_b.append(r)
+            i=q
+        number_in_b.reverse()
+        better_format = int("".join(map(str, number_in_b)))
```

Comment:
If you turn the digits into strings, concatenate them, and then through `int` it is changed back into a decimal number again. This is not what you want.